### PR TITLE
add pylbm package

### PR DESCRIPTION
**Pull request by gouarin**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pylbm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - future
+    - colorlog
+
+test:
+  imports:
+    - pylbm
+
+about:
+  home: http://pylbm.readthedocs.io
+  license: BSD 3-Clause
```
Comment (chrisburr) [inline recipes/pylbm/meta.yaml]: The SPDX identifier has two dashes:
```suggestion
  license: BSD-3-Clause
```

File: recipes/pylbm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/pylbm/pylbm/archive/{{ version }}.tar.gz
+  sha256: 9a9263a5f14fbbd6a65b73c7e904c1add789f3334228f9535309d533a104b552
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true  # [win]
```
Comment (chrisburr) [inline recipes/pylbm/meta.yaml]: Can this be `noarch`? It doesn't matter if Windows fails due to missing dependencies
```suggestion
  noarch: python
```
If not you need to remove the version constraint from Python and use `skip: true  # [win or py<34]` instead

Comment (gouarin) [reply recipes/pylbm/meta.yaml]: this package can be `noarch` with `Python>=3.4`. I removed it due to the windows build failure.

**Comment by gouarin** [comment]
@chrisburr the noarch doesn't work for me. I have updated the recipe. I can see the 0.4.1 available on conda-forge but the command line

```
conda install -c conda-forge pylbm 
```

doesn't find any package.

How can I regenerate CI files to build on linux and mac OS X platforms ?

**Comment by gouarin** [comment]
@chrisburr I found my mistake !

**Pull request merged**